mmtbx/ions/identify.py

```diff
@@ -1957,8 +1960,8 @@ def check_ion_environment(self,
                 # XXX probably just O
               self.bad_coords[identity].append(contact)
               inaccuracies.add(self.BAD_COORD_RESIDUE)
-        elif (cmp(0, mmtbx.ions.server.get_charge(contact.atom)) ==
-              cmp(0, ion_params.charge)):
+        elif (_sign(mmtbx.ions.server.get_charge(contact.atom)) ==
+              _sign(ion_params.charge)):
```

Comment:
Duplicate `cmp`
Change the import to `from libtbx.math_utils import cmp` so nothing else needs to be changed.